pkg/client/patch.go

```diff
@@ -72,3 +77,21 @@ func (s *mergeFromPatch) Data(obj runtime.Object) ([]byte, error) {
 func MergeFrom(obj runtime.Object) Patch {
 	return &mergeFromPatch{obj}
 }
+
+// applyPatch uses server-side apply to patch the object.
+type applyPatch struct{}
```

Comment:
You don't use the `var _ SomeInterface = someImpl{}` pattern? I'm assuming this implements something that I don't know

Reply:
See the `Patch` interface above